woocommerce-gateway-paypal-express-checkout.php

```diff
@@ -25,6 +25,10 @@
 	exit; // Exit if accessed directly
 }
 
+if ( ! defined( 'WC_GATEWAY_PPEC_VERSION' ) ) {
+	define( 'WC_GATEWAY_PPEC_VERSION', '1.2.1' );
```

Comment:
So for this if you recall, we agreed to just define it without first checking as we want it to break as soon as possible.